subsys/bluetooth/controller/ll_sw/ull_llcp.c

```diff
@@ -619,6 +619,23 @@ void ull_cp_conn_param_req_neg_reply(struct ll_conn *conn)
 	}
 }
 
+uint8_t ull_cp_conn_enc_encryption_paused(struct ll_conn *conn)
```

Comment:
Why the double "enc" ?

Reply:
to keep with the pattern - ull_cp_conn_enc_<....> because it is enc procedure related

Reply:
ahh, sorry I see - it is not quite so - I'll rename to ull_cp_encryption_paused()